assets/js/components/user-input/UserInputSelectOptions.js

```diff
@@ -155,11 +155,20 @@ export default function UserInputSelectOptions( {
 	const ListComponent = max === 1 ? Radio : Checkbox;
 
 	const items = Object.keys( options ).map( ( optionSlug ) => {
+		if ( 'sell_products_or_service' === optionSlug ) {
+			return false;
+		}
+
 		const props = {
 			id: `${ slug }-${ optionSlug }`,
 			value: optionSlug,
 			description: descriptions?.[ optionSlug ],
-			checked: values.includes( optionSlug ),
+			checked:
+				USER_INPUT_QUESTIONS_PURPOSE === slug &&
+				values.includes( 'sell_products_or_service' ) &&
+				'sell_products' === optionSlug
+					? true
+					: values.includes( optionSlug ),
```

Comment:
I believe this change is not longer because we changed the selected value from `sell_products_or_service` to `sell_products` in the `UserInputPreview` component, right?

Reply:
Thanks @eugene-manuilov this has been resolved in the latest commit.